allennlp/tests/modules/self_attentive_sentence_encoder_test.py

```diff
@@ -0,0 +1,54 @@
+# pylint: disable=no-self-use,invalid-name
+import numpy
+import torch
+
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.modules.self_attentive_sentence_encoder import SelfAttentiveSentenceEncoder
+
+
+class TestSelfAttentiveSentenceEncoder(AllenNlpTestCase):
+
+    def test_forward_and_dimensions(self):
+        self_attentive_encoder = SelfAttentiveSentenceEncoder(3, 4, 5, False)
```

Comment:
could you use keyword arguments here, that way someone reading the test can know what 3 4 and 5 are